Makefile

```diff
@@ -339,6 +339,14 @@ format:
 	@go run github.com/golangci/golangci-lint/cmd/golangci-lint run ./... --fix
 	@go run mvdan.cc/gofumpt -l -w x/ app/ ante/ tests/
 
+mdlint:
+	@echo "--> Running markdown linter"
+	@docker run -v $(PWD):/workdir ghcr.io/igorshubovych/markdownlint-cli:latest "**/*.md" --disable MD013 MD029 MD004 MD033
+
+markdown:
+	@echo "--> Running markdown linter"
+	@docker run -v $(PWD):/workdir ghcr.io/igorshubovych/markdownlint-cli:latest "**/*.md" --disable MD013 MD029 MD004 MD033 --fix
```

Comment:
Err, this should be using the main file

Reply:
I'm assuming you mean pointing the container to the most recent release directly instead of `latest` – let me know if the most recent change doesn't address your point

Reply:
No, I mean the .markdownlint file